### PR TITLE
Fixed localization

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
Requires the theme changes at https://github.com/go-gitea/theme/pull/16

**Comment by lunny** [comment]
build failed?

**Comment by tboerger** [comment]
See the comment above

**Comment by tboerger** [comment]
Build fixed

**Comment by Bwko** [comment]
LGTM

File: content/index.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+date: "2016-11-08T16:00:00+02:00"
+title: "欢迎"
+weight: 10
+toc: false
+draft: false
+type: "default"
+---
+
+# Gitea - Git with a cup of tea
```
Comment (bkcsoft) [inline content/index.zh-cn.md:10]: Maybe @lunny can translate this before merging?

**Review by bkcsoft — CHANGES_REQUESTED** [review]
If I press "Chinese" I expect to get things in Chinese 🙂

File: content/index.zh-cn.md (hunk 1 above)
Comment (bkcsoft) [reply content/index.zh-cn.md:10]: Otherwise I can translate to Swedish tomorrow 😛

Comment (tboerger) [reply content/index.zh-cn.md:10]: Let lunny do that in a followup pr, currently we got something to see that the i18n stuff works

**Comment by metalmatze** [comment]
LGTM

**Review by bkcsoft — APPROVED** [review]

File: content/index.zh-cn.md (hunk 1 above)
Comment (lunny) [reply content/index.zh-cn.md:10]: https://github.com/go-gitea/website/pull/15
and how to enter the right language according browser setting?

**Pull request merged**